<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticipantWindow</class>
 <widget class="QWidget" name="ParticipantWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParticipantWindow</string>
  </property>
  <widget class="QListWidget" name="questionList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="scoreLine">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Score</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Answer:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="answerLine"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
